platform/admin/core/admin-core-logviewer/README.md

```diff
@@ -0,0 +1,62 @@
+<!--
+/*
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+-->
+# Admin UI LogViewer
+
+## Directory Structure
+### src/main/webapp
+The top level of this directory contains index.html and any other javascript files that do not pertain to react or redux.
++ `actions`: Redux actions files.
++ `backend`: Files relevant to retrieving log lists from the logging endpoint.
++ `components`: Files that return React view components.
++ `reducers`: Redux reducer files.
++ `css`: CSS files.
+
+### src/test/js
+Contains javascript unit tests.
+
+## Tests and Development
+### Running Unit tests
+To run tests, do:
+
+    npm test
+
+### Hot-Reloading changes
+The npm package `hotloadify` is contained as a dev dependency inside `package.json`, so setting up a hot reloading environment for development can be run by typing:
+
+    npm start
+    
+If pulling from a live instance of DDF, a proxy will need to be used. This is an example of setting up a hot reload environment with a proxy from the top level of this bundle:
+
+    hotloadify src/main/webapp/js/index.js --proxy='https://localhost:8993' --open
+    
+## What is expected from the backend?
+The backend is expected to send a JSON object formatted as follows:
+    
+    {
+        "request": {
+            "mbean": "org.codice.ddf.logging.platform.LoggingServiceBean:service=logging-service",
```

Comment:
The code that hits the backend goes to the right url, this is just an example of the response we get back from it. I'll go ahead and change this to avoid confusion though.